Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM registry.redhat.io/openshift4/ose-tools-rhel8:v4.13.0-202306070816.p0.g05d83ef.assembly.stream
+
+COPY --chown=root:root --chmod=644 data/ca-trust/* /etc/pki/ca-trust/source/anchors
```

Comment:
Why not `--cmod=444`?

Reply:
this chawn and the build process in general has nothing to do with the container being root or non-root. In practice the build is running non-root when I run it locally simply because that is how my podman is configured.

Setting 444 will make things harder for no good reason: If you got root in the container you can chmod and write, if you don't then the root write bit doesn't give you anything.